docs/ros/ros/installation/controller.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Joystick Controller Pairing
+sidebar_label: Joystick Controller
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Joystick controllers are used for teleoperation; they allow you to remotely drive the Jackal, whether it is a physical Jackal robot,
+or a simulated Jackal. The following instructions below detail how to pair different controllers to the Jackal’s computer; however,
+these instructions can also be used to pair these controllers to your own computer.
+
+### Logitech F710 Controller
```

Comment:
Let's deprecate the Logitech controller and remove it from our documentation. I think we want to avoid using this controller any longer due to safety concerns, right?

Reply:
What should we do about customers who have logitech controllers and want to upgrade to Humble? Some of them may not have bluetooth available on the robot computer.

Reply:
I guess they can still use it, but it will be undocumented and unsupported. That said, looking at the documentation, there isn't actually any commands they need to run for Logitech, so it should "just work", right?

Reply:
We will have to add instructions to set the controller type in the robot.yaml file. But otherwise it should just work.